_assets/js/ta-selectors.js

```diff
@@ -46,8 +47,13 @@ dropdownButton.addEventListener('click', () => {
 });
 
 window.onload = function () {
-  selectorState = getSearchParam('filters').split(';');
-  // If selector state is empty string, set to empty array
+  // If there is something in local storage, grab that state and use it:
+  if (accessStorage('filters') !== null && getSearchParam('filters') === '') {
```

Comment:
Could this be simplified to 

```
if (accessStorage('filters') && !getSearchParam('filters'))
```
to catch more cases?

Reply:
Unfortunately no. The search params have to be set to match the state of the checkboxes on page load, so if the state is empty, the search params have to be set to an empty string.